src/main/java/Command/UnMarkCommand.java

```diff
@@ -0,0 +1,28 @@
+package Command;
+
+import Ping.Task;
+import Ping.TaskList;
+import Ping.UI;
+
+/**
+ * This class is used to unmark a task
+ */
+public class UnMarkCommand extends Command {

```

Comment:
Here, "Unmark" seems to be one word to me! Maybe you could rename this class as `UnmarkCommand`?